doc-content/enterprise-only/employee-rostering/optashift-ER-building-commandline-db-proc.adoc

```diff
@@ -17,12 +17,20 @@ If you use the command line to build the employee rostering starter application
 +
 [source,bash]
 ----
-java -jar optaweb-employee-rostering-standalone-*-exec.jar --spring.profiles.active=production
-spring.datasource.url=<DATABASE_URL> --spring.datasource.username=<DATABASE_USER> --spring.datasource.password=<DATABASE_PASSWORD>
+java -jar quarkus-app/quarkus-run.jar --
+java -Dquarkus.datasource.username=<DATABASE_USER> -Dquarkus.datasource.password=<DATABASE_PASSWORD> -Dquarkus.datasource.jdbc.url=<DATABASE_URL> -jar optaweb-employee-rostering-standalone/target/quarkus-app/quarkus-run.jar
```

Comment:
Either remove the first one or separate it from the second one. The first one starts it using an in-memory database (good for trying out the application); the second one connects to an existing database.

```suggestion
java -jar quarkus-app/quarkus-run.jar
----

or to connect it to an existing database:
----
java -Dquarkus.datasource.username=<DATABASE_USER> -Dquarkus.datasource.password=<DATABASE_PASSWORD> -Dquarkus.datasource.jdbc.url=<DATABASE_URL> -jar optaweb-employee-rostering-standalone/target/quarkus-app/quarkus-run.jar
```